modules/nix/nufetch.nu

```diff
@@ -0,0 +1,10 @@
+export def main [] { 
+
+{
+"kernel": (uname | get kernel-release), 
```

Comment:
instead of calling an external command, you could call:
1. for kernel version `$nu.os-info.kernel_version`
2. for nushell version `$env.NU_VERSION`
3. for uptime `(sys host).uptime` which returns a nushell duration